app/pods/publications/publication/proofs/index/template.hbs

```diff
@@ -38,41 +38,43 @@
       data-test-route-publications---proofs-panel-body
       class="auk-u-bg-alt"
     >
-      {{#each @model as |event|}}
-        {{#if event.isShown}}
-          {{#if event.isRequestActivity}}
-            <Utils::Timeline::Event
-              @badgeIcon="mail"
-              @badgeSkin="primary"
-              @event={{event}}
-              as |event|
-            >
-              <Publications::Publication::RequestActivityPanel
-                @title={{t "requested-proof"}}
-                @requestActivity={{event.activity}}
-                @isDisabledDelete={{event.activity.proofingActivity.isFinished}}
-                @onDeleteRequest={{perform this.deleteRequest event.activity}}
-              />
-            </Utils::Timeline::Event>
-          {{else if event.isProofingActivity}}
-            <Utils::Timeline::Event
-              @badgeIcon="check"
-              @badgeSkin="success"
-              @event={{event}}
-              as |event|
-            >
-              <Publications::Publication::Proofs::ProofReceivedPanel
-                @proofingActivity={{event.activity}}
-                @publicationFlow={{this.publicationFlow}}
-                @proofPrintCorrector={{this.publicationSubcase.proofPrintCorrector}}
-                @onEditProofingActivity={{perform this.editProofingActivity}}
-                @onCreatePublicationRequest={{perform
-                  this.savePublicationRequest
-                }}
-                @isCreatePublicationRequestDisabled={{this.isCreatePublicationRequestDisabled}}
-              />
-            </Utils::Timeline::Event>
-          {{/if}}
+      {{#each this.shownActivities as |event|}}
+        {{#if event.isRequestActivity}}
+          <Utils::Timeline::Event
+            @badgeIcon="mail"
+            @badgeSkin="primary"
+            @event={{event}}
+            as |event|
+          >
+            <Publications::Publication::RequestActivityPanel
+              @title={{t "requested-proof"}}
+              @requestActivity={{event.activity}}
+              @isDisabledDelete={{event.activity.proofingActivity.isFinished}}
+              @onDeleteRequest={{perform this.deleteRequest event.activity}}
+            />
+          </Utils::Timeline::Event>
+        {{else if event.isProofingActivity}}
+          <Utils::Timeline::Event
+            @badgeIcon="check"
+            @badgeSkin="success"
+            @event={{event}}
+            as |event|
+          >
+            <Publications::Publication::Proofs::ProofReceivedPanel
+              @proofingActivity={{event.activity}}
+              @publicationFlow={{this.publicationFlow}}
+              @proofPrintCorrector={{this.publicationSubcase.proofPrintCorrector}}
+              @onEditProofingActivity={{perform this.editProofingActivity}}
+              @onDeletePiece={{if
+                event.canDeletePieces
+                (perform this.deleteReceivedPiece event)
+              }}
```

Comment:
Vraagje hierbij (wellicht door mijn gebrek aan ervaring met ember): de delete-knop in `PublicationDocumentsList` wordt enkel getoond als `@onDeletePiece` gebonden/truthy is, toch? Zie https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/9e8f704ae15705b8d46ea157f567dbfe03a4ecd2/app/components/publications/publication/publication-documents-list.hbs#L42-L50

Als je `@onDeletePiece` hier op deze manier instelt, gaat hij dan de delete-knop ook tonen als `event.canDeletePieces` falsy is?

Reply:
De delete-knop wordt niet getoond als `@onDeletePiece` false/undefined is. (Ik zou dit eigenlijk aanpassen, zodat de knop in dat geval gedisabled is. Dat lijkt me UX-gewijs correcter en meer in lijn met andere knoppen, maar da's een ander punt.)